scripts/prerelease/generate_prerelease_script.py

```diff
@@ -79,7 +79,8 @@ def main(argv=sys.argv[1:]):
         type=_repository_name_and_type_and_url_and_branch,
         default=[],
         metavar='REPO_NAME:REPO_TYPE:REPO_URL:BRANCH_OR_TAG_NAME',
-        help='The name, type, url and branch / tag name of a repository')
+        help='The name, type, url and branch / tag name of a repository.' +
+             'e.g. "common_tutorials:git:https://github.com/ros/common_tutorials:pullrequest-1"')
```

Comment:
Updated.
And `+` removed as suggested at https://github.com/ros-infrastructure/ros_buildfarm/pull/358#discussion_r93327818.